app/views/users/new.html.haml

```diff
@@ -1,13 +1,46 @@
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+<script type="text/javascript">
+Stripe.setPublishableKey("#{Rails.env.production? ? ENV['STRIPE_LIVE_PUB_KEY'] : 'pk_test_5jJnBEWP58uma9eoJZsbahsq'}");
```

Comment:
You should be using something like `figaro` to store this value for different environments, rather than putting a conditional in your view template.